test-forms/google_password_manager_search.html

```diff
@@ -0,0 +1,31 @@
+<div class="QJfW3">
+    <div class="rFrNMe L9di L3JtQc zKHdkd u3bW4e" jscontroller="pxq3x" jsaction="clickonly:KjsqPd; focus:Jt1EX; blur:fpfTEe; input:Lg5SV" jsshadow="" jsname="btrqz">
+       <div class="aCsJod oJeWuf">
+          <div class="aXBtI I0VJ4d Wic03c">
+             <span jsslot="" class="A37UZe qgcB3c iHd5yb">
+                <div class="VfPpkd-Bz112c-LgbsSe yHy1rc eT1oJ QDwDD mN1ivc BeY8ie" jscontroller="nKuFpb" jsaction="click:cOuCgd; mousedown:UX7yZ; mouseup:lbsD7e; mouseenter:tfO1Yc; mouseleave:JywGue; touchstart:p6p2H; touchmove:FwuNnf; touchend:yfqBxc; touchcancel:JMtRjd; focus:AHmuwe; blur:O22p3e; contextmenu:mg9Pef;mlnRJb:fLiPzd;" data-idom-class="yHy1rc eT1oJ QDwDD mN1ivc BeY8ie" style="--mdc-ripple-fg-size: 28px; --mdc-ripple-fg-scale: 1.7142857142857142; --mdc-ripple-left: 10px; --mdc-ripple-top: 10px;">
+                   <div jsname="s3Eaab" class="VfPpkd-Bz112c-Jh9lGc"></div>
+                   <i class="google-material-icons" aria-hidden="true">arrow_back</i><a class="WpHeLc VfPpkd-mRLv6" href="?ep=1&amp;f.sid=5221294659451512461" aria-label="Back" jsname="hSRGPd" data-focusid="6"></a>
+                   <div class="VfPpkd-Bz112c-J1Ukfc-LhBDec"></div>
+                </div>
+             </span>
+             <div class="Xb9hP">
+                <input type="text" class="whsOnd zHQkBf" jsname="YPqjbf" autocomplete="off" tabindex="0" aria-label="Search passwords" aria-disabled="false" autofocus="" data-initial-value="" data-ddg-inputtype="unknown">
+                <div jsname="LwH6nd" class="ndJi5d snByac" aria-hidden="true" data-manual-scoring="unknown">Search passwords</div>
```

Comment:
Watch out, you kept the inputtype attribute and added the manual-scoring to a div 🙂.